### PR TITLE
Small Updates

### PR DESCRIPTION
**Pull request by wilwade**
# Goal
The goal of this PR is several small updates.

Part of #255 and #339 

# Discussion
See PR inline comments

# Checklist
- [x] Chain spec updated

### CODE REVIEW

File: node/service/src/chain_spec/frequency.rs [hunk 1]
```diff
@@ -44,8 +44,7 @@ pub mod frequency_mainnet_keys {
 pub fn frequency() -> ChainSpec {
 	let properties =
 		get_properties(FREQUENCY_TOKEN, TOKEN_DECIMALS as u32, SS58Prefix::get().into());
-	// TODO need a paraid here for the frequency chain
-	let para_id: ParaId = 999.into();
+	let para_id: ParaId = 2091.into();
```
Comment (wilwade) [inline node/service/src/chain_spec/frequency.rs:47]: Expected final paraId

File: node/service/src/chain_spec/frequency.rs [hunk 2]
```diff
@@ -164,7 +163,7 @@ pub fn frequency() -> ChainSpec {
 			"/dns/1.boot.frequency.xyz/tcp/30333/p2p/12D3KooWCW8d7Yz2d3Jcb49rWcNppRNEs1K2NZitCpPtrHSQb6dw".parse().unwrap(),
 		],
 		// Telemetry
-		TelemetryEndpoints::new(vec![("wss://telemetry.polkadot.io/submit/".into(), 0)]).ok(),
+		TelemetryEndpoints::new(vec![("wss://telemetry.polkadot.io/submit/".into(), 0), ("wss://telemetry.frequency.xyz/submit/".into(), 0)]).ok(),
```
Comment (wilwade) [inline node/service/src/chain_spec/frequency.rs:166]: We will also be running our own telemetry in addition to the official one

File: node/service/src/chain_spec/frequency_rococo.rs [hunk 3]
```diff
@@ -24,7 +24,7 @@ pub mod public_testnet_keys {
 	pub const COLLATOR_2_SR25519: &str =
 		"0x58025fbd92bcc65fc2e054b623d10a8558662ae545a9718e26815024a2433545";
 	pub const ROCOCO_FRQ_SUDO: &str =
-		"0xa4c16e3ce8b11a0e43102bc4a718e9f031df2d0f61b1b03031b2f45ed664f853";
+		"0xccca4a5b784105460c5466cbb8d11b34f29ffcf6c725d07b65940e697763763c";
```
Comment (wilwade) [inline node/service/src/chain_spec/frequency_rococo.rs:27]: Updating sudo account

File: node/service/src/chain_spec/frequency_rococo.rs [hunk 4]
```diff
@@ -157,7 +157,9 @@ pub fn frequency_rococo_testnet() -> ChainSpec {
 			)
 		},
 		// Bootnodes
-		Vec::new(),
+		vec![
+			"/dns/0.boot.rococo.frequency.xyz/tcp/30333/p2p/5CkhzgoWc21tKSYATXhba2Rj1Nhu6subu8FEfbxPk9GFjGqd".parse().unwrap(),
```
Comment (wilwade) [inline node/service/src/chain_spec/frequency_rococo.rs:161]: Eventual rococo bootnode

File: runtime/frequency/src/lib.rs [hunk 5]
```diff
@@ -80,6 +80,7 @@ impl Contains<Call> for BaseCallFilter {
 			Call::TechnicalCommittee(..) => true,
 			Call::Council(..) => true,
 			Call::Democracy(..) => true,
+			Call::Session(..) => true,
```
Comment (wilwade) [inline runtime/frequency/src/lib.rs:83]: We need session so that collators can rotate their keys

**Review by enddynayn — APPROVED** [review]
🙌

**Pull request merged**